README.md

```diff
@@ -1,17 +1,20 @@
 # Homebrew webpage
 
-The webpage uses [jekyll](https://github.com/mojombo/jekyll). The template for the index is at layouts/index.html.
+The webpage uses [jekyll](https://github.com/mojombo/jekyll). The template for
+the index is at `_layouts/index.html`.
```

Comment:
I’m sorry I don’t understand what you want here. Do you want me put each sentence on its own line? I broke the line to keep the text under 80 chars but this doesn’t change the way how the text is displayed.